docs/en_US/Compressor/Framework.md

```diff
@@ -1,6 +1,16 @@
-# Design Doc
+# Customize A New Compression Algorithm
 
```

Comment:
From the Overview.md
> Details about how model compression framework works can be found in here.

this doc seems to explain "how model compression framework works", right? But now, it seems turn into a tutorial??

if we'd like to explain how the framework works, we might want to add a component diagram or workflow diagram to ease the reading..